test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -67,21 +98,29 @@ TEST_F(OrderingServiceTest, ValidWhenProposalSizeStrategy) {
   EXPECT_CALL(*wsv, getLedgerPeers()).WillRepeatedly(Return(std::vector<Peer>{
       peer}));
 
-  service = std::make_shared<OrderingServiceImpl>(wsv, 5, 1000, loop);
+  const size_t num = 5;
+  service = std::make_shared<OrderingServiceImpl>(wsv, num, 1000, loop);
 
   EXPECT_CALL(*fake_gate, SendProposal(_, _, _)).Times(2);
+  ON_CALL(*fake_gate, SendProposal(_, _, _))
+      .WillByDefault(Invoke([this](auto, auto, auto) {
+        cv.std::condition_variable::notify_one();
+        std::cout<<"+\n";
```

Comment:
change cout with logger. For creating logger use `testLog` function.

Reply:
Ah, missed it before committing
I'll better leave more meaningful message, thanks!